### DISCUSSION TITLE
Hi. I'm new to git-hub community.

### DISCUSSION THREAD

**Discussion by Matrixlaw** [community/community#82964] [New to GitHub]
I've been an on/off programmer as such for around 35years.
o CP/M [8080]. Coding in Basic, ASM
o Cobol,Pascal,OOP, C, C++ Unix,Linux,VMS, CSS,HTML, Excel Basic, Excel Macro scripts [8086, 80386,80486, Pentium, AMD64,today etc...]
o Code CAD scripts for TurboCAD, Solidworks CAD, Autocad...
o Dropped away from  C, C++ and prefer Borland Pascal, OOP, Free Pascal,Delphi and now converting to Larzarus.
o Moving away from misanthropic "Microsoft Windows" and focusing on past years using the Linux platform. 
The above should give the Git-hub community the general over view...

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Matrixlaw 🎆! 

We're glad you're here and can't wait to see where your GitHub journey takes you. To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). You can also check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
